diff --git a/dist/index.js b/dist/index.js
index 7f31afcc47e0e98ee7d4debc471c4f55e7453b89..43bfa940f5ab6fdd206771b39179833e25fcf493 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -5,7 +5,7 @@ import { useKeyboardControls, TransformControls, useHelper } from "@react-three/
 import { clamp } from "three/src/math/MathUtils";
 import { create } from "zustand";
 import { subscribeWithSelector } from "zustand/middleware";
-import { StaticGeometryGenerator, computeBoundsTree, disposeBoundsTree, acceleratedRaycast, MeshBVHHelper, SAH } from "three-mesh-bvh";
+import { StaticGeometryGenerator, computeBoundsTree, disposeBoundsTree, acceleratedRaycast, BVHHelper, SAH } from "three-mesh-bvh";
 /*!
  * BVHEcctrl
  * https://github.com/pmndrs/BVHEcctrl
@@ -660,7 +660,7 @@ const StaticCollider = forwardRef(({
     verbose: false,
     setBoundingBox: true,
     maxDepth: 40,
-    maxLeafTris: 10,
+    maxLeafSize: 10,
     indirect: false
   },
   ...props
@@ -739,7 +739,7 @@ const StaticCollider = forwardRef(({
       mergedMesh.current.userData.excludeCollisionCheck = excludeCollisionCheck;
     }
   }, [props.visible, bvhName, friction, restitution, excludeFloatHit, excludeCollisionCheck]);
-  useHelper(debug && mergedMesh, MeshBVHHelper);
+  useHelper(debug && mergedMesh, BVHHelper);
   return /* @__PURE__ */ React.createElement("group", { ref: colliderRef, ...props, dispose: null }, children);
 });
 const StaticCollider$1 = React.memo(StaticCollider);
@@ -764,7 +764,7 @@ const KinematicCollider = forwardRef(({
     verbose: false,
     setBoundingBox: true,
     maxDepth: 40,
-    maxLeafTris: 10,
+    maxLeafSize: 10,
     indirect: false
   },
   ...props
@@ -869,7 +869,7 @@ const KinematicCollider = forwardRef(({
       mergedMesh.current.userData.excludeCollisionCheck = excludeCollisionCheck;
     }
   }, [props.visible, bvhName, active, friction, restitution, excludeFloatHit, excludeCollisionCheck]);
-  useHelper(debug && mergedMesh, MeshBVHHelper);
+  useHelper(debug && mergedMesh, BVHHelper);
   useFrame((state, delta) => {
     if (!mergedMesh.current || !colliderRef.current || !active)
       return;
@@ -939,7 +939,7 @@ const InstancedStaticCollider = forwardRef(({
     verbose: false,
     setBoundingBox: true,
     maxDepth: 40,
-    maxLeafTris: 10,
+    maxLeafSize: 10,
     indirect: false
   },
   ...props
@@ -1015,7 +1015,7 @@ const InstancedStaticCollider = forwardRef(({
       if (bvhHelper.current) {
         bvhHelper.current.visible = debug;
       } else {
-        bvhHelper.current = new MeshBVHHelper(mergedMesh.current, 20);
+        bvhHelper.current = new BVHHelper(mergedMesh.current, 20);
         bvhHelper.current.visible = debug;
         scene.add(bvhHelper.current);
       }
